### PR TITLE
[BEAM-1288] Clean *.pyc files with mvn clean.

### PR DESCRIPTION
**Pull request by aaltay**
Added maven-clean-plugin configuration to clean artifacts generated by
other phases of mvn.

Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by aaltay** [comment]
R: @dhalperi

File: sdks/python/pom.xml [hunk 1]
```diff
@@ -56,6 +56,22 @@
     </pluginManagement>
     <plugins>
       <plugin>
+        <artifactId>maven-clean-plugin</artifactId>
+        <version>3.0.0</version>
```
Comment (dhalperi) [inline sdks/python/pom.xml]: can all this be moved into the root `pom.xml` `pluginManagement` section?

Specifically, the `version` should be inherited, but I don't think any of this necessarily applies only to the `sdks`  module. (E.g., we might have examples, runners, etc. someday.)

Comment (aaltay) [inline sdks/python/pom.xml]: Done. PTAL.

**Comment by dhalperi** [comment]
LGTM, will merge once green. Thanks!

**Comment by dhalperi** [comment]
Merged, thanks.

**Comment by aaltay** [comment]
Thank you.

**Pull request closed without merging**